sdk/RELEASE.md

```diff
@@ -18,7 +20,7 @@
 
 ## Major Features and Improvements
 
-* N/A
+* Improve CLI experience for archiving experiments, managing recurring runs and listing resources [\#6934](https://github.com/kubeflow/pipelines/pull/6934)
```

Comment:
You can revert this. 1.8.9 has been cut already. This change will go into 1.8.10 later (we'll move it around by the time we do the release).

Reply:
Ah apologies about that, I thought I'd fixed that! Reverted now